### PR TITLE
adding loinc2hpo

### PR DESCRIPTION
**Pull request by jmcmurry**

### CODE REVIEW

File: loinc2hpo/htaccess.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Options +FollowSymLinks
+
+RewriteEngine on
+RewriteRule ^$ https://github.com/monarch-initiative/loinc2hpo/blob/master/README.md [R=302,L]
+RewriteRule ^annotations$ https://github.com/TheJacksonLaboratory/loinc2hpoAnnotation/blob/master/Data/TSVSingleFile/annotations.tsv [R=302, L]
```
Comment (davidlehn) [inline loinc2hpo/htaccess.txt]: Did you want to link to the raw TSV file or the github ui?

**Comment by jmcmurry** [comment]
Thanks for merging; just wanted to have a heads up as to when redirection will be live?

**Comment by davidlehn** [comment]
Whoops, that `htaccess.txt` file needs to be named `.htaccess`.  Can you make a PR for that rename?  It goes live as soon as it's merged.

**Comment by jmcmurry** [comment]
whoops indeed; will fix

**Comment by dgarijo** [comment]
My bad, I didn't see the file extension :)

**Pull request merged**